### PR TITLE
Glossary model exchange

### PR DESCRIPTION
**Pull request by clagms**
This PR summarizes the changes made while adding terms to the glossary in the Model Exchange section.
This is part of a larger effort to update our glossary.

Some extra changes are made, because:
1. They are too small to have in a separate PR
2. I am convinced that they are small improvements to the text

Other issues that I found but did not know how to fix, are summarized in 
https://github.com/modelica/fmi-standard/issues/898
https://github.com/modelica/fmi-standard/issues/926

### CODE REVIEW

**Comment by andreas-junghanns** [comment]
Works - thanks!

**Comment by andreas-junghanns** [comment]
Why is this PR waiting?

**Comment by clagms** [comment]
> Why is this PR waiting?

Because I'm terribly busy with something else :p

File: docs/3_1_model_exchange_math.adoc [hunk 1]
```diff
@@ -208,15 +208,15 @@ All discrete-time variables are fixed during this phase and the corresponding di
 ===== Event Mode
 This mode is used to compute new values for all continuous-time variables, as well as for all discrete-time variables that are activated at the current event instant latexmath:[t], given the values of the variables from the previous instant latexmath:[{}^{\bullet}t].
 This is performed by solving algebraic equations consisting of all continuous-time and all active discrete-time equations.
-In FMI 2.0 there is no mechanism that the FMU can provide the information whether a discrete-time variable is active or is not active (is not computed) at an event instant.
+In FMI, there is no mechanism that the FMU can provide the information whether a discrete-time variable is active or is not active (is not computed) at an event instant.
```
Comment (chrbertsch) [inline docs/3_1_model_exchange_math.adoc:211]: Is this true for clocks and clocked variables? But this is not the intention of this PR, so I create a new ticket https://github.com/modelica/fmi-standard/issues/1045

Comment (andreas-junghanns) [reply docs/3_1_model_exchange_math.adoc]: @chrbertsch : I tried to fix this in a later commit (117a288) to this branch - can you check and if you approve, merge this PR?

**Pull request merged**